addons/skin.estuary/xml/DialogMusicInfo.xml

```diff
@@ -414,11 +445,105 @@
 				</control>
 				<include content="LeftRightArrows">
 					<param name="list_id" value="50" />
-					<param name="left_posx" value="562" />
-					<param name="right_posx" value="1780" />
+					<param name="left_posx" value="565" />
+					<param name="right_posx" value="1790" />
 					<param name="posy" value="300" />
 					<param name="visible" value="true" />
 				</include>
+				<control type="group">
+					<control type="image">
+						<left>1315</left>
+						<top>474</top>
+						<width>450</width>
+						<height>390</height>
+						<aspectratio>scale</aspectratio>
+						<texture colordiffuse="AAFFFFFF">colors/black.png</texture>
+					</control>
+					<control type="group">
+						<left>1275</left>
+						<top>480</top>
+						<control type="list" id="41">
+							<left>42</left>
+							<top>0</top>
+							<width>430</width>
+							<height>385</height>
+							<onleft>130</onleft>
+							<onright>130</onright>
+							<onup>50</onup>
+							<ondown>9000</ondown>
+							<pagecontrol/>
+							<scrolltime>200</scrolltime>
+							<itemlayout height="35">
+								<control type="label">
+									<left>10</left>
+									<top>0</top>
+									<width>150</width>
+									<height>30</height>
+									<font>font27_narrow</font>
+									<label>$INFO[ListItem.Label,[COLOR button_focus], [/COLOR]]</label>
+								</control>
+								<control type="label">
+									<left>170</left>
+									<width>240</width>
+									<height>30</height>
+									<font>font27_narrow</font>
+									<label>$INFO[ListItem.Label2]</label>
+								</control>
+							</itemlayout>
+							<focusedlayout height="35">
+								<control type="image">
+									<left>0</left>
+									<top>1</top>
+									<width>430</width>
+									<height>33</height>
+									<texture border="8" colordiffuse="button_focus">lists/focus.png</texture>
```

Comment:
the `border` attribute is not needed with the new texture, could you please drop it?

Reply:
done